### ISSUE TITLE
Sanctum Feedback

### ISSUE THREAD

**Issue by cretueusebiu** [cretueusebiu/laravel-vue-spa#325]
I want to replace [JWT](https://github.com/tymondesigns/jwt-auth) with [Sanctum](https://laravel.com/docs/8.x/sanctum) however it's not that straight forward...

Right now with JWT everything is stateless, meaning that you can consume any api routes from anywhere.
But moving to Sanctum some api routes don't make sense to be api routes, but rather just web routes (login, logout, ouath, verify email).
Because if use the api with a mobile application it would be stateless and would have other api routes for [issuing tokens](https://laravel.com/docs/8.x/sanctum#issuing-mobile-api-tokens) for example.

So the question is do you use the api routes for something more that just the web app ? 
And does it make sense to keep the api routes with Sanctum ? Or just move everything to use the default session guard ?

**Comment by ghost** [comment]
> So the question is do you use the api routes for something more that just the web app ?

yes, data import / export with random apps,
and initiator may be at random side (my app or another).

**Comment by OzanKurt** [comment]
Is there a branch of your tests about this topic where we can play around and maybe help?

**Comment by unsa-cdn** [comment]
Hi, what is the state of this? I think Sanctum is a must if you plan on getting up to date with Laravel 9

**Comment by cretueusebiu** [comment]
@unsa-cdn Yeah.. I'll replace jwt with Sanctum very soon. I was also thinking to use Breeze maybe ? Since now it supports spa stack.
